### DISCUSSION TITLE
Geolocalisation for non-GPS images

### DISCUSSION THREAD

**Discussion by elito** [LycheeOrg/Lychee#2336] [Ideas]
Hi,
i've just discovered lychee and i'm very enthousiaste. Thank for the work done.

I've noted that images from my nikon camera have no GPS data is it possible to add them in lychee.

If not have you a app suggestion to do this ?

**Comment by ildyria** [comment]
Moved to Ideas.
I think this should be doable.

To edit:
https://github.com/LycheeOrg/Lychee/blob/master/resources/js/data/panel/photoFormPanel.ts
https://github.com/LycheeOrg/Lychee/blob/master/resources/views/components/gallery/photo/properties.blade.php
https://github.com/LycheeOrg/Lychee/blob/master/app/Livewire/Forms/PhotoUpdateForm.php

**Reply by elito** [reply]
Thank's i will check this, but i'm not a good developper so maybe i will wait for it
